hieradata/class/staging/postgresql_primary.yaml

```diff
@@ -1 +1 @@
-govuk_postgresql::server::max_connections: 120
+govuk_postgresql::server::max_connections: 200
```

Comment:
If we're setting this everywhere we may as well just set it as the default in the manifest itself.